assets/js/modules/pagespeed-insights/datastore/report.js

```diff
@@ -96,6 +99,20 @@ const baseSelectors = {
 
 		return reports[ `${ strategy }::${ url }` ];
 	},
+
+	/**
+	 * Gets a PageSpeed Insights error for the given strategy and URL if it has been received.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state    Data store's state.
+	 * @param {string} url      URL used for generating the report.
+	 * @param {string} strategy Strategy used for generating the report.
+	 * @return {(Object|undefined)} A PageSpeed Insights error; `undefined` if not thrown.
+	 */
+	getReportError: createRegistrySelector( ( select ) => ( state, url, strategy ) => {
+		return select( STORE_NAME ).getErrorForSelector( 'getReport', { url, strategy } );
+	} ),
```

Comment:
See other comment, let's remove this. Also, if we kept it, it would need to use `[ url, strategy ]`.

Reply:
Removed.